examples/torchtune/qwen2_5/qwen2.5-1.5B-with-alpaca.ipynb

```diff
@@ -0,0 +1,276 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7c49a6d5",
+   "metadata": {},
+   "source": [
+    "# Fine-tune Qwe2.5-1.5B with Alpaca Dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e3b15eff",
+   "metadata": {},
+   "source": [
+    "This example demonstrates how to fine-tune Qwen2.5-1.5B model with the Alpaca Dataset using TorchTune `BuiltinTrainer` from Kubeflow Trainer SDK.\n",
+    "\n",
+    "This notebooks walks you through the prerequisites of using TorchTune `BuiltinTrainer` from Kubeflow Trainer SDK, and how to submit TrainJob to bootstrap the fine-tuning workflow.\n",
+    "\n",
+    "Qwen2.5-1.5B: https://huggingface.co/Qwen/Qwen2.5-1.5B-Instruct\n",
+    "\n",
+    "Alpaca Dataset: https://huggingface.co/datasets/tatsu-lab/alpaca"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ed4a60eb",
+   "metadata": {},
+   "source": [
+    "## Install the Kubeflow SDK\n",
+    "\n",
+    "You need to install the Kubeflow SDK to interact with Kubeflow Trainer APIs:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "288ec515",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install git+https://github.com/kubeflow/sdk.git@main"
```

Comment:
Please can you comment this line, since we install SDK as part of CI. Also, since we publish the SDK, you can just pip install kubeflow.
```suggestion
    "# !pip install kubeflow"
```

Reply:
done